### PR TITLE
fix: doc of renamed fb_function

### PR DESCRIPTION
**Pull request by hsawaji**
update doc and comment

- create_file -> create
- open_file   -> open

### CODE REVIEW

File: doc/telescope-file-browser.txt [hunk 1]
```diff
@@ -78,7 +78,7 @@ fb_picker.file_browser({opts})                      *fb_picker.file_browser()*
                                         `vim.fn.expanded` automatically
                                         (default: vim.loop.cwd())
         {cwd}               (string)    dir to browse folders from,
-                                        `vim.fn.expanded` automatically
```
Comment (fdschmidt93) [inline doc/telescope-file-browser.txt:81]: Typically, CI should trigger automatically to generate `doc/telescope-file-browser.txt`, but for some reason this only so far worked for my own PRs.

Can you please also remove the extra space in `picker.lua`?

Then it's ok to merge as lua mirror doc files perfectly.

**Review by fdschmidt93 — APPROVED** [review]
Many thanks for your PR :)

I'll merge after the required change in `picker.lua` of removing the extra space.

File: doc/telescope-file-browser.txt (hunk 1 above)
Comment (hsawaji) [reply doc/telescope-file-browser.txt:81]: I removed the extra space!
70f2e11fbf70082fbb5dbcb7c140f5cf0b361af4

Comment (fdschmidt93) [reply doc/telescope-file-browser.txt:81]: Thanks :)

**Pull request merged**